docs/src/further_topics/ugrid/partner_packages.rst

```diff
@@ -0,0 +1,99 @@
+.. _ugrid partners:
+
+Iris' Mesh Partner Packages
+****************************
+Python is an easy to use language and has formed a very strong collaborative
+scientific community, which is why Iris is written in Python. *Performant*
+Python relies on calls down to C, which is ideal for structured grid work since
```

Comment:
Since ESMF uses Fortran, this could be better as "Python relies on calls down to low level languages like C".

Reply:
be5f51bc6a379de5f020ea418f0c88f57ecf5527